README.md

```diff
@@ -33,13 +33,21 @@ final createdTestCase = await TestCase.create(
 );
 ```
 
+### Delete Test case
+```dart
+/// Get TestCase by ID
+final testCase = await TestCase.get(1);
+
+await testCase.delete();
+```
+
```

Comment:
@tshedor isn't this is what we need in README?

Reply:
Yes